### PR TITLE
Fixed issue 5256, improved 'occluded' visualization

### LINKED ISSUE #5256: Skeleton Track: Hide keypoints does not take effect immediately
### My actions before raising this issue
- [X] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [X] Searched [past issues](/issues)

When using the new skeleton annotation feature, when we hide a keypoint, it does not take effect immediately.

### Expected Behaviour

When clicking the "Switch Outside Property" on a keypoint, it should hide right away.

### Current Behaviour

When clicking the "Switch Outside Property" on a keypoint,  it is need to go one frame forward and one frame back to see it disappear.

### Context

It is confusing because sometimes we don't know if we correctly set the keypoint outside.

### Your Environment

- Git hash commit (`git log -1`): 46b88c11ab2bbb4989365899196bf4157ee94b90
- Docker version `docker version` (e.g. Docker 17.0.05): 20.10.21
- Are you using Docker Swarm or Kubernetes? No
- Operating System and version (e.g. Linux, Windows, MacOS): Ubuntu 22.04

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
Resolved #5256

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [x] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Review by klakhov — COMMENTED** [review]
I see now dashes apply instantly😄
Also found some inconvinient behavior. When im trying to turn occluded off on one part, its not updated:
skeleton-occluded

**Comment by klakhov** [comment]
Also there is red rest_api test, `TestTaskUsecases.test_cant_create_task_with_no_data` doesn't seem to be related to the patch. @kirill-sizov could you take a look?

**Comment by kirill-sizov** [comment]
> Also there is red rest_api test, `TestTaskUsecases.test_cant_create_task_with_no_data` doesn't seem to be related to the patch. @kirill-sizov could you take a look?

Yes, this problem is not related with this PR

File: cvat-canvas/src/typescript/canvasView.ts [hunk 1]
```diff
@@ -1916,10 +1916,11 @@ export class CanvasViewImpl implements CanvasView, Listener {
             }
 
             if (drawnState.occluded !== state.occluded) {
+                const instance = state.shapeType === 'points' ? this.svgShapes[clientID].remember('_selectHandler').nested : shape;
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat-canvas/src/typescript/canvasView.ts:1919]: I do not really understand what you mean

**Review by klakhov — APPROVED** [review]

**Comment by bsekachev** [comment]
Failed because of:
image

**Pull request merged**